config/config/common-config.yaml

```diff
@@ -613,3 +613,13 @@ networkPolicies:
     scIngress:
       ips: # ip(s) to sc loadbalancer if available, otherwise sc worker nodes
         - set-me
```

Comment:
```suggestion
    scIngress:
      ips: # ip(s) to sc loadbalancer if available, otherwise sc worker nodes
        - set-me
     wcIngress:
      # ip(s) to wc loadbalancer if available, otherwise wc worker nodes
      ips: 
        - set-me
```